2-mandatory/3-financial-times.js

```diff
@@ -15,27 +25,68 @@ function potentialHeadlines(allArticleTitles) {
 */
 function titleWithFewestWords(allArticleTitles) {
     // TODO
+
+    let ascesndingTitles = allArticleTitles.sort((a, b) => a.length - b.length);
```

Comment:
This passes the test and is a clever way to get the shortest title... however, what are we actually looking for? Are we looking for the shortest title or are we looking for the title with the fewest words?